src/RestaurantCardsContainer.jsx

```diff
@@ -8,35 +8,57 @@ class RestaurantCardsContainer extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      restaurants: []
+      restaurants: [],
+      error: null,
     };
   }
 
   componentDidMount() {
-    fetch('https://learnreact.avicndugu.repl.co/restaurants.json')
-      .then(res => res.json())
+    fetch('https://learnreact.avicndugu.repl.co/resstaurants.json')
+      .then((response) => {
+        if (response.ok) {
+          return response.json();
+        } else {
+          throw Error(response.status);
+        }
+      })
       .then(
-        data => this.setState({
-          restaurants: data.restaurants
-        })
+        (data) => {
+          this.setState({
+            restaurants: data.restaurants,
+          });
+        },
+        (error) => {
+          this.setState({
+            error: error,
+          });
+        }
       );
   }
 
   render() {
-    const { restaurants } = this.state;
-    return (
-      <Container>
-        <Row xs={1} md={3}>
-        {
-          restaurants.map((restaurant, index) => (
-            <Col>
-              <RestaurantCard key={index} name={restaurant.restaurantName} address={restaurant.address} hours={restaurant.hours} />
-            </Col>
-          ))
-        }
-        </Row>
-      </Container>
-    );
+    const { restaurants, error } = this.state;
+    if (error) {
+      return <div>Error code/ message: {error.message}</div>;
+    } else {
+      return (
+        <Container>
+          <Row xs={1} md={3}>
+            {restaurants.map((restaurant, index) => (
+              <Col>
+                <p>{error}</p>
+                <RestaurantCard
+                  key={index}
+                  name={restaurant.restaurantName}
+                  address={restaurant.address}
+                  hours={restaurant.hours}
+                />
+              </Col>

```

Comment:
`key={index}` should be passed to `<Col>` at line 48.

Reply:
I have moved key={index} to <Col>